src/app/fetch-status.service.ts

```diff
@@ -65,46 +64,40 @@ export class FetchStatusService {
     });
   }
 
-  private startFetching(statusMap: any): void {
+  startFetching(statusMap: Map<string, BehaviorSubject<Status>>): void {
```

Comment:
Shouldn't this still be private, we don't want this to be called outside of the service, right?